src/test/scala/nl.knaw.dans.easy.properties/app/DeleterSpec.scala

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (C) 2019 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.properties.app
+
+import java.util.UUID
+
+import cats.scalatest.EitherValues
+import cats.syntax.either._
+import nl.knaw.dans.easy.properties.fixture.{ DatabaseDataFixture, DatabaseFixture, FileSystemSupport, TestSupportFixture }
+
+import scala.util.{ Failure, Try }
+
+class DeleterSpec extends TestSupportFixture
+  with FileSystemSupport
+  with DatabaseFixture
+  with DatabaseDataFixture
+  with EitherValues {
+
+  private val uuid5: UUID = UUID.fromString("00000000-0000-0000-0000-000000000005") // exists
+  private val uuid6: UUID = UUID.fromString("00000000-0000-0000-0000-000000000006") // does not exist
+
+  "delete" should "fail with a foreign key violation" in {
+    repository.deposits.deleteBy(Seq(uuid5)).leftValue.msg shouldBe
+      "integrity constraint violation: foreign key no action; SYS_FK_10153 table: STATE"
+  }
+
+  it should "fail with a NullPointerException" in {
```

Comment:
removed these tests in my review (https://github.com/DANS-KNAW-jp/easy-deposit-properties/pull/3)

Reply:
A toString is now used in another context and throws an exception rather than return a left
https://github.com/DANS-KNAW/easy-deposit-properties/blob/c9a24ef11fa059fc2f5f78a6295fe1a8d0c2947e/src/main/scala/nl.knaw.dans.easy.properties/app/repository/sql/SQLDeletable.scala#L39-L40
https://github.com/DANS-KNAW/easy-deposit-properties/blob/38f4c918765df899a3fc244f8c11e1d190d80be3/src/main/scala/nl.knaw.dans.easy.properties/app/repository/sql/SQLDeletable.scala#L39-L44